juniper/src/integrations/chrono.rs

```diff
@@ -157,17 +151,17 @@ mod test {
 
     #[test]
     fn naivedate_from_input_value() {
-        naivedate_test("1996-12-19T16:39:57-08:00", 1996, 12, 19);
+        naivedate_test("1996-12-19", 1996, 12, 19);
     }
 
     #[test]
-    fn naivedate_from_input_value_with_fractional_seconds() {
-        naivedate_test("1996-12-19T16:39:57.005-08:00", 1996, 12, 19);
-    }
+    fn naivedate_serialization() {
+        use types::base::GraphQLType;
 
-    #[test]
-    fn naivedate_from_input_value_with_z_timezone() {
-        naivedate_test("1996-12-19T16:39:57Z", 1996, 12, 19);
+        let date = NaiveDate::from_ymd(2000, 1, 2);
+        let value = date.resolve(&(), None, &executor::Executor);
```

Comment:
I figured out what to pass for the first two args, but don't know about the third.
Any help on this? Thanks

Reply:
@LegNeato @theduke can I get your help on this please? Thanks

Reply:
@sporto I did a PR to your PR with some integration tests. Feel free to structure them another way.